src/icespeak/settings.py

```diff
@@ -164,6 +168,12 @@ class AzureKey(BaseModel, frozen=True):
     region: SecretStr
 
 
+class OpenAIKey(BaseModel, frozen=True):
+    "Format of an API key for OpenAI."
+
+    OPENAI_API_KEY: SecretStr
```

Comment:
Tiny tiny nitpick, should the attribute name maybe be in lowercase, e.g. `api_key: SecretStr` - just for consistency with the other key models?